src/gc-tls.h

```diff
@@ -49,7 +49,7 @@ typedef struct {
 
 typedef struct {
     _Atomic(int64_t) allocd;
-    _Atomic(int64_t) pool_live_bytes;
+    _Atomic(int64_t) small_objects_live_bytes;
     _Atomic(uint64_t) malloc;
     _Atomic(uint64_t) realloc;
     _Atomic(uint64_t) poolalloc;
```

Comment:
Missed this one (and related code) perhaps?
```suggestion
    _Atomic(uint64_t) smallalloc;
```